docs/usage.rst

```diff
@@ -4,10 +4,10 @@ Usage
 .. code:: python
 
     import vcr
-    import urllib2
```

Comment:
hmm, we should probably lint our docs

https://pypi.org/project/flake8-rst/

Reply:
should i do the linting?